.dockerignore

```diff
@@ -1,3 +1,4 @@
+.git/
```

Comment:
this might cause issues with some of the other docker images that are doing `-e` installs